docs/conf.py

```diff
@@ -126,8 +126,9 @@
     "doc_module": "pyvista",
     "image_scrapers": ('pyvista', 'matplotlib'),
     'first_notebook_cell': ("%matplotlib inline\n"
-                            "from pyvista import set_plot_theme\n"
-                            "set_plot_theme('document')"),
+                            "from pyvista import set_jupyter_backend, set_plot_theme\n"
+                            "set_plot_theme('document')\n"
+                            "set_jupyter_backend('ipyvtklink')"),
```

Comment:
FYI, is this okay?

Reply:
Yes.  It's always confusing for a new user why it's not working with dynamic plots.

Reply:
In fact, I'd like to set it as default for the entire module.  Can we make `ipyvtklink` a hard dependency?

Reply:
> In fact, I'd like to set it as default for the entire module. Can we make ipyvtklink a hard dependency?

Might not work out if this requires ipython at some point in the install...  Maybe warn the user?

Reply:
I'm hesitant to add it as a hard dependency for those that want to use PyVista outside of ipython and don't want to install that stack as well. I think soft is the way to go with a clear error message or warning when folks try to use it.

Let's set it as the default and if it isn't available and the user is in Jupyter, throw a warning and fallback to the static plot

Reply:
Sounds good.  I already have people asking (both internally and externally) for interactive notebook plots.  Giving them a warning would help them with the installation.

Reply:
See the latest changes. Now the code will always fallback to showing a static output:

![Screen Shot 2021-04-14 at 4 12 24 PM](https://user-images.githubusercontent.com/22067021/114786953-3522be00-9d3c-11eb-981e-513e9ae1983b.png)

Reply:
Perfect.  This will make sense to new and existing users.